test/meta_logger_test.exs

```diff
@@ -64,8 +57,56 @@ defmodule MetaLoggerTest do
     end
   end)
 
+  describe "warn/2" do
```

Comment:
Why adding this block instead of just making line 8 equal to `~w(debug error info warn warning)a`? If the reason is the assertion that doesn't expected `warn`, I think you could iterate over something like this:
```elixir
# {level(), level_label()}
[
  {:debug, "debug"},
  {:error, "error"},
  {:info, "info"},
  {:warn, "warning"},
  {:warning, "warning"},
]
```

What do you think?

Reply:
Thought about that and I was inclined to do exactly that, but I decided to isolate the `warn` tests since it will be removed in a future release.

The current approach makes clear the detachment and you can just remove the entire `describe` block when the time comes.

With the suggestion, the future dev would have to either move back to the previous code or keep the `{level(), level_label()}` logic that will not be relevant for the tests anymore.